spec/services/iiif_thumbnail_path_service_spec.rb

```diff
@@ -7,6 +7,8 @@
   before do
     allow(ActiveFedora::Base).to receive(:find).with('s1784k724').and_return(file_set)
     allow(file_set).to receive_messages(original_file: file, id: 's1784k724')
+    # https://github.com/projecthydra/active_fedora/issues/1251
```

Comment:
Not a blocker, but another way to do this would have been `let(:file_set) { mock_model(FileSet, id: 's1784k724') }`